test/JDBC/upgrade/14_14/schedule

```diff
@@ -465,4 +465,4 @@ binary-datatype-operators
 cast-varchar-to-time
 xml_exist-before-16_5
 BABEL-CASE_EXPR-before-16_5-or-15_9
-
```

Comment:
Nit: we should keep the newline at end of file.

Reply:
In some files, there were 2 newlines. So, removed one.